client/my-sites/plans/jetpack-plans/jetpack-complete-page/components/cta-buttons/style.scss

```diff
@@ -0,0 +1,40 @@
+.jetpack-complete-page__cta-buttons {
+	display: inline-flex;
+	align-items: center;
+	justify-content: center;
+	width: 100%;
+	& > * {
+		flex: 1;
+	}
+	@include breakpoint-deprecated( "<960px" ) {
+		flex-direction: column;
+	}
```

Comment:
Conventionally we try to do "mobile first," so larger screens are the ones that end up with breakpoints. The equivalent would be something like this:

```scss
flex-direction: column;

@include breakpoint-deprecated( '>960px' ) {
	flex-direction: row;
}
```